### PR TITLE
Fix #308 vertical alignment of header buttons

### LINKED ISSUE #308: Graphical glitch on top link list
I see the link list in this way:
Schermata da 2019-04-16 22-14-13
with a floating white bar at the bottom.

### PR DESCRIPTION
**Pull request by FredBonux**
Closes #308

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -148,7 +148,7 @@ DEPENDENCIES
   tzinfo-data
 
 RUBY VERSION
-   ruby 2.6.0p0
+   ruby 2.6.3p62
```
Comment (libremente) [inline Gemfile.lock]: Why? This is out of scope.

File: Gemfile.lock [hunk 2]
```diff
@@ -148,7 +148,7 @@ DEPENDENCIES
...
 
 BUNDLED WITH
-   2.0.1
+   2.0.2
```
Comment (libremente) [inline Gemfile.lock]: Same as above.

**Comment by FredBonux** [comment]
@libremente ooops, didn’t noticed that. Sorry

**Comment by libremente** [comment]
Hi @FredBonux, did you have the opportunity to fix those issues? Thanks!

**Comment by FredBonux** [comment]
@libremente It should be fixed now.

**Comment by libremente** [comment]
Awesome @FredBonux, I like it! 
Let's wait for @sebbalex review. Thanks!

**Comment by libremente** [comment]
Merged, thankd @FredBonux! 🎉
